docs: explain monetization change flow and remove rental pricing model as an option (#2333)
apify/apify-docs: sources/academy/build-and-publish/apify-store-basics/how_actor_monetization_works.md

### Changing monetization

Monetization changes fall into two categories:

> - _Non-significant changes_ (take effect immediately): decreasing prices, removing events, updating descriptions

Reviewer: ```suggestion
- Non-significant changes that take effect immediately: decreasing prices, removing events, updating descriptions
```

> - _Significant changes_ (require a 14-day notice period): increasing prices, adding new paid events, changing the pricing model

Reviewer: ```suggestion
- Significant changes that require a 14-day notice period: increasing prices, adding new paid events, changing the pricing model
```

> When you submit a significant change, your users are notified and the current pricing stays active for 14 days. You can only make significant changes **once per month** per Actor, and you **cannot cancel** a planned change on your own - you need to contact [Apify support](https://apify.com/contact).

Reviewer 2: Rephrase to get rid of passive voice & change bold to emphasis. Bold is for UI elements only

```suggestion
When you submit a significant change, your users receive a notification and the current pricing stays active for 14 days. You can only make significant changes _once per month_ per Actor, and you _cannot cancel_ a planned change on your own - you need to contact [Apify support](https://apify.com/contact).
```

> When you submit a significant change, your users receive a notification and the current pricing stays active for 14 days. You can only make significant changes _once per month_ per Actor, and you _cannot cancel_ a planned change on your own - you need to contact [Apify support](https://apify.com/contact).

Reviewer: ```suggestion
When you submit a significant change, your users receive a notification and the current pricing stays active for 14 days. You can only make significant changes once per month per Actor. To cancel a planned change, [contact Apify support](https://apify.com/contact).
```

Learn more about what makes a good readme here: [How to create an Actor README](/academy/actor-marketing-playbook/actor-basics/how-to-create-an-actor-readme)

> ### Choose pay-per-event (PPE)

Reviewer 2: I think we can remove it from heading it's on the first sentence of the section as well

```suggestion
### Choose pay-per-event
```

Author: I think we can remove this paragraph, it's not possible to choose any pricing model other than PPE now.

Pay-per-event (PPE) gives you the most flexibility and growth potential. You can charge for any meaningful event your Actor performs (for example, page scraped, browser page opened, or an external API call). This makes costs predictable for users, lets you model value precisely, and is fully compatible with AI and MCP-based integrations.

Rental pricing is being sunset (see the [rental documentation](/platform/actors/publishing/monetize/rental) for the timeline) and is no longer recommended for new Actors.

:::tip Additional benefits

Actors that implement PPE receive additional benefits, including increased visibility in Apify Store and enhanced discoverability.

:::

> To estimate pricing, run a few test runs and review the statistics in the Actor [**Analytics**](https://console.apify.com/actors?tab=analytics) tab.

Reviewer 2: Is this intentionally a repeat of L151?

Author: it was because of the rental warning, I just didn't realize both were so close to each other, removed the whole paragraph
